chat.go

```diff
@@ -13,6 +13,25 @@ import (
 // about a chat command that's taking long to execute.
 var ChatCmdTimeout = 10 * time.Second
 
+// Exec handles a string as if it where send by a client.
```

Comment:
Aaaaargh english
And why is it called Exec?
What about DoChatMsg or something like that?